content/course-specific-material/introR-Reston-Aug17.Rmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Reston Intro to R Training"
+date: "2017-08-27"
+slug: "introR-Reston-Aug17"
+output: USGSmarkdowntemplates::hugoTraining_course
+parent: Course Specific Material
+---
+August 27-29 in Reston, VA
+
+### Installation
+
+See [Before the Workshop](/intro-curriculum/Before) for information on what software should be installed prior to the course. 
```

Comment:
oof forgot to mention that this can change - we now have a top-level page for installation instructions: `[R Installation Instructions](/installr)`